lib/block_view_types.go

```diff
@@ -1073,6 +1116,12 @@ func (op *UtxoOperation) RawEncodeWithoutMetadata(blockHeight uint64, skipMetada
 		data = append(data, EncodeToBytes(blockHeight, entry, skipMetadata...)...)
 	}
 
+	// PrevUserAssociationEntry
+	data = append(data, EncodeToBytes(blockHeight, op.PrevUserAssociationEntry, skipMetadata...)...)
```

Comment:
Just randomly found this lol but I think this needs to be gated by an encoder migration. Otherwise there might be problems decoding the existing entries in the db without re-syncing. I've done something similar in #412 for UtxoOperation encode/decode.

Reply:
Good call, @AeonSw4n. :+1: